django_ninja_crudl/service.py

```
@@ -1,362 +1,362 @@
-            field_name
-            for field_name in self.list_schema.model_fields
-            if field_name not in related_models + many_to_many_models + property_fields
-        ]
-
-        all_fields: list[str] = non_related_fields + related_fields
-
-        qs = qs.select_related(*related_models).prefetch_related(
-            *many_to_many_models,
-        )  # To avoid N+1 queries
-        return qs.values(*all_fields)
+# """CRUDL API base class."""
```

Ruff (0.8.2)

Missing docstring in public module

(D100)